### PR TITLE
IKeyboardToolTip implementation

### PR DESCRIPTION
**Pull request by panda2u**
IKeyboardToolTip implementation added

### CODE REVIEW

**Comment by zsd4yr** [comment]
@panda2u this is a change to a public class; we ask that you please open a corresponding issue and allow the API suggestion to go through our regular review process

**Comment by danmosemsft** [comment]
@panda2u also, could you please move large cleanup changes out into a separate PR? It seems like the functional changes are in the first two commits, then there is a huge cleanup change, which inevitably now has huge conflicts. That way they can be reviewed and committed independently.

**Comment by danmosemsft** [comment]
Particularly since cleanup changes can often be committed as quickly as they are reviewed (and a good thing due to quickly getting conflicts), whereas functional changes may need discussion.

File: src/System.Windows.Forms/src/System/Windows/Forms/Button.cs [hunk 1]
```diff
@@ -392,15 +392,15 @@ public class Button : ButtonBase, IButtonControl {
...
+                case NativeMethods.WM_REFLECT + NativeMethods.WM_COMMAND:
                     if (NativeMethods.Util.HIWORD(m.WParam) == NativeMethods.BN_CLICKED) {
                         Debug.Assert(!GetStyle(ControlStyles.UserPaint), "Shouldn't get BN_CLICKED when UserPaint");
                         if (!ValidationCancelled) {
                             OnClick(EventArgs.Empty);
                         }                        
                     }
                     break;
-                case Interop.WindowMessages.WM_ERASEBKGND:
+                case NativeMethods.WM_ERASEBKGND:
```
Comment (hughbe) [inline src/System.Windows.Forms/src/System/Windows/Forms/Button.cs]: This doesn't look deliberate :)

**Comment by danmosemsft** [comment]
@panda2u we appreciate your change, but I think we need formatting changes broken out (into PR's that are not too large and self coherent if possible) and any changes to public API will need agreement in an issue before a PR for those. Meantime since it's been quiet in this PR I will close this and you are very welcome to submit new ones as described. Thanks.

**Pull request closed without merging**